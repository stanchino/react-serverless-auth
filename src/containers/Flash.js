import { connect } from "react-redux";

import FlashMessages from "../components/FlashMessages";

const mapStateToProps = state => ({
    flash: state.auth.flash
});

export default connect(mapStateToProps)(FlashMessages);