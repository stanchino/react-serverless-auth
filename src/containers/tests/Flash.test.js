import React from "react";
import renderer from "react-test-renderer";
import createStore from "redux-mock-store";

import Flash from "../Flash";

const store = createStore()({ auth: { flash: {} } });

describe("Flash", () =>{
    it("matches the snapshot", () => {
        expect(renderer.create(<Flash store={store} />).toJSON()).toMatchSnapshot();
    });
});