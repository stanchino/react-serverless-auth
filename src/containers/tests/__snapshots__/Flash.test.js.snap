// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flash matches the snapshot 1`] = `
<div
  className="flash"
/>
`;
